unified-runtime/test/conformance/device/urDeviceGetSelected.cpp

```diff
@@ -6,14 +6,37 @@
 
 #include <uur/fixtures.h>
 
+#ifdef _MSC_VER
```

Comment:
Could this be pulled out into a header file somewhere just in case other tests in future might want to use it?

Reply:
We can do that if/when we need to. I'd rather not encourage this behavior TBH.